config/webpack/webpack.config.main.prod.js

```diff
@@ -17,7 +17,7 @@ export default merge.smart(baseConfig, {
 
   // 'main.js' in root
   output: {
-    path: __dirname,
```

Comment:
`__dirname` will return the directory containing this file and not the root folder, so by using `process.cwd()` we make sure that the paths are relative to the folder where the node command is run from.